app/src/utils/utils.ts

```diff
@@ -426,29 +426,39 @@ export const shuffle = (unshuffled: Grant[] | GrantRound[]) => {
 export const recursiveGraphFetch = async (
```

Comment:
One small comment is that it looks like the function's natspec comments are out of date

Reply:
Good catch - I've refactored so that we don't have to pass in the `lastId` and deleted the aforementioned comment 🙌